flake8_trio/__init__.py

```diff
@@ -53,6 +74,44 @@ def cst_parse_module_native(source: str) -> cst.Module:
     return mod
 
 
+def main():
+    parser = ArgumentParser(prog="flake8_trio")
+    parser.add_argument(
+        nargs="*",
+        metavar="file",
+        dest="files",
+        help="Files(s) to format, instead of autodetection.",
+    )
+    Plugin.add_options(parser)
+    args = parser.parse_args()
+    Plugin.parse_options(args)
+    if args.files:
+        # TODO: go through subdirectories if directory/ies specified
+        all_filenames = args.files
```

Comment:
🤷‍♂️, I'm pretty happy to say that you have to pass exact files if you're passing files.

Reply:
I figured, since that's how shed does it, but I would like to support as much of flake8's functionality as possible to ease any transition from running [name of this project] as a flake8 plugin to running it standalone. Ofc flake8 is kinda crazy so I won't bother supporting everything (e.g. `--extend-ignore`, `--ignore`, `--select`, `--extend-select` I think kinda just makes everything more complicated), but I think supporting directories shouldn't be very complicated and doesn't have much downsides.